app/components/layer-wrapper/style.scss

```diff
@@ -47,4 +47,6 @@
     }
 }
 
-
+.margin-20{
```

Comment:
Can you use the helper classes before doign this? we have m-md or mg-lt. They are not exactly 20: https://centerforopenscience.github.io/osf-style/style.html#layout-helper